src/data/helpers.tsx

```diff
@@ -1,12 +1,31 @@
 import * as R from 'ramda'
 
-const importAll = (r: any) => r.keys().map(r)
-/**
- * Implements ```import data from './*.json'```
- *
- * From https://webpack.js.org/guides/dependency-management/#context-module-api
- */
-export const importData: <T>(
+export const importSingleFile: <T extends {} = any>(
   path: string,
-  includeSubdirs: boolean,
-) => T[] = R.pipe(R.partialRight(require.context, [/\.json$/]), importAll)
+) => Promise<T> = path =>
+  import(path)
+    .then(x => x.default)
+    .catch(x => console.error(x))
+
+export const importFiles: <T extends {} = any>(
+  directory: string,
+  files: string[],
+) => Promise<T[]> = R.curry((directory, files) =>
+  Promise.all(
+    R.pipe(
+      R.map(file => `${directory}${file}.json`),
+      R.map(importSingleFile),
+    )(files),
+  ),
+)
+
+export const expandCollection: <S>(
```

Comment:
```suggestion
export const expandReferences: <S>(
```
(not sure if this name is better)